contrib/templates/data-engineering/base/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -0,0 +1,50 @@
+# This is a Databricks asset bundle definition for {{.project_name}}.
+# See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
+bundle:
+  name: {{.project_name}}
+
+include:
+  - assets/*.yml
```

Comment:
So far, we have placed all yml files under `resources/`. What is our guidance for when to use `assets` and when to use `resources`?

We could also keep the pipeline code in `assets` and move the yml under `resources`. That would be in line with our current guidance.

cc @pietern

Reply:
The thing here is that `src` and `resources` are no longer separate. So it's not just what we call a "resource" in DABs.